src/include/function/string/operations/regexp_extract_all_operation.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "common/re2_regex.h"
+#include "common/types/ku_list.h"
+#include "common/types/ku_string.h"
+#include "common/vector/value_vector.h"
```

Comment:
I think value_vector_utils.h includes value_vector and value_vector also includes ku_string.h, so you can probably remove the include for value_vector, ku_string.h, ku_list.h.